rllib/BUILD

```diff
@@ -46,8 +46,7 @@
 # Additional tags are:
 # - "team:ml": Indicating that all tests in this file are the responsibility of
 #   the ML Team.
-# - "needs_gpu": Indicating that a test needs to have a GPU in order to run.
-# - "gpu": Indicating that a test may (but doesn't have to) be run in the GPU
+# - "gpu_rllib_X": Indicating that a test needs to be run in the GPU
```

Comment:
What does the "X" mean?

Reply:
The gpu runner buildkite tags for rllib are named gpu_rllib_1 and gpu_rllib_2. I thought c would be a nice placeholder for a number.